eng/common/templates/job/job.yml

```diff
@@ -217,7 +217,7 @@ jobs:
       displayName: Publish TRX Test Results
       inputs:
         testResultsFormat: 'VSTest'
-        testResultsFiles: '*.trx' 
+        testResultsFiles: '*.xml'
```

Comment:
Is this a break in Arcade or do we need to somehow backport this to Arcade?

Reply:
Yep, I'm just testing if this is the right fix